src/tap.h

```diff
@@ -27,6 +27,10 @@ class TAP : public QObject
 public:
     explicit TAP(QObject *parent = 0);
     void send(const QByteArray &data);
+    void mtuChanged(int mtu);
```

Comment:
The Qt convention is that xChanged is used for signal and slots would have names like: onXChanged... or in this case since it's more of an action on the tap interface, I would name it changeMtu()

Naming things is hard... :)